doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -0,0 +1,148 @@
+[id='test-scenarios-comparison-legacy-new-ref']
+
+= Feature comparison of legacy and new test scenario designer
+
+{PRODUCT} currently supports both the new *Test Scenarios* designer and the former *Test Scenarios (Legacy)* designer.
```

Comment:
Suggested rewrite:
{PRODUCT} supports both the legacy and new test scenarios designer. 

//I'd suggest removing currently word. Also, we do not need to capitalize and bold Test Scenarios since we're not referring to the UI.

Reply:
Done.